addons/mail/static/src/models/audio.js

```diff
@@ -0,0 +1,118 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Audio',
+    lifecycleHooks: {
+        _created() {
+            if (window.Audio === undefined) {
+                return;
+            }
+            this.update({ audio: new window.Audio() });
+        },
```

Comment:
If `Audio` is deleted while `WebAudio` is being played, the record will be deleted but `WebAudio` will still play.
We deal with short audio samples, and we don't really delete them, so it's not a problem right now.
But conceptually it would make sense to `audio.pause()` if it's being played, so this `WebAudio` is properly garbage collectable at the same time as deletion of record.